test/react_memo_test.dart

```diff
@@ -58,64 +111,62 @@ main() {
         reason: 'test setup sanity check');
   }
 
-  group('memo', () {
```

Comment:
Best reviewed [with whitespace-agnostic diff](https://github.com/cleandart/react-dart/pull/275/files?w=1#diff-3c4e4fda8814c627565672f04c4999fdL61); this actually didn't change